source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -9,12 +9,16 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/readpref"
 )
 
-// Connection URI
-const uri = "mongodb://user:pass@sample.host:27017/?maxPoolSize=20&w=majority"
+// Paste your connection string here
+const uri = "<connection string>"
```

Comment:
S: Different docs use different terms for this: "connection string", "connection URI", etc. Might be good to bring up to the team to settle on a standard term (doesn't necessarily need to be before this is merged).